src/pages/ArticlePage.jsx

```diff
@@ -88,14 +142,39 @@ function ArticlePage() {
             </div>
     
             <div className="comments-section">
-            <h3>Comments</h3>
+            <h3>Comments:</h3>
+
+
+            <form onSubmit={handleCommentSubmit}>
+                <input
+                    type="text"
+                    placeholder="Your name"
+                    value={username}
+                    onChange={(e) => setUsername(e.target.value)}
+                    required
+                />
```

Comment:
Ideally the user should not need to type their username every time they wish to post a comment so this could just be hardcoded instead. Until some form of login is introduced.